src/theme/ItaliaTheme/_main.scss

```
@@ -151,6 +151,9 @@ iframe {
         color: $primary-text;
         font-weight: 700;
       }
+      &.link-anchorlink-theme {
+        color: #fff !important;
```

```suggestion
        color: $primary-text !important;
```

non forziamolo a bianco, se il sito ha un colore primario particolare per cui il testo sul colore primario non è bianco, non va bene. Prova a vedere se così ottieni lo stesso risultato.